RELEASE/scripts/autoscend/auto_island_war.ash

```diff
@@ -101,79 +100,96 @@ boolean warAdventure()
 
 boolean L12_getOutfit()
 {
-	if (internalQuestStatus("questL12War") < 0 || internalQuestStatus("questL12War") > 0)
+	if (internalQuestStatus("questL12War") != 0)
 	{
 		return false;
 	}
 
+	// noncombat when adventuring at The Hippy Camp (Verge of War)
+	set_property("choiceAdventure139", "3");	//fight a War Hippy (space) cadet for outfit pieces
+	set_property("choiceAdventure140", "3");	//fight a War Hippy drill sergeant for outfit pieces
+	set_property("choiceAdventure141", "1");	//if wearing [Frat Boy Ensemble] get 50 mysticality
+	set_property("choiceAdventure142", "3");	//if wearing [Frat Warrior Fatigues] start the war or skip adventure

```

Comment:
unnecessary here. You're also setting this in `L12_startwar()` where it's relevant.

Reply:
change made